Galway 2020/js/custom.js

```diff
@@ -30,7 +30,7 @@ function removeClass(element, name) {
   element.className = arr1.join(" ");
 }
 
-// Add active class to the current button (highlight it)
+
 let btnContainer = document.getElementById("myBtnContainer");
 let btns = btnContainer.getElementsByClassName("btn");
 for (let i = 0; i < btns.length; i++) {

```

Comment:
I can see javascript is functioning the way it should be, the filter is working well.